### ISSUE TITLE
Add support for Discord Auth

### ISSUE THREAD

**Issue by cemkoker** [thedevdojo/auth#101]
It would be great to add support for Discord Authentication.

**Comment by skdishansachin** [comment]
I'm not entirely certain about this because I haven't tested it myself. But based on the docs, it looks like you can use different providers like...

> These are all part of [Laravel Socialite](https://devdojo.com/auth/docs/config/social-providers/). If you want to use additional providers that are not listed above, you'll need to install another package called [Socialite Providers](https://socialiteproviders.com/).

Link to the [docs.](https://devdojo.com/auth/docs/config/social-providers/#socialite-providers-package)

**Comment by cemkoker** [comment]
Yes indeed, but I was asking if it could be built-in as it is one of the most used ones.

**Comment by skdishansachin** [comment]
I believe it's up to @tnylea to make the call. I think, for now, anyone who needs the Discord authentication provider can install and set it up themselves. If we receive more requests to add Discord as a built-in option, we can think about it. I don't mean to dismiss your idea, but the currently supported authentication providers are common and reliable. Adding too many built-in providers wouldn't make much sense, you know? Thanks for your idea, but ultimately, @tnylea will have to decide.

**Comment by cemkoker** [comment]
Ultimately ... yes, I was just surprised to see TikTok (as we know we all use TikTok to authenticate most of our web services) ... and not Discord. It would be great to have Discord added that's all. Thanks !
